### DISCUSSION TITLE
Error when running Optimization example

### DISCUSSION THREAD

**Discussion by carolinafernandezp** [jonescompneurolab/hnn-core#576] [Q&A] [answered]
Hi,

I'm running the [Optimization example](https://jonescompneurolab.github.io/hnn-core/stable/auto_examples/howto/plot_optimize_evoked.html#sphx-glr-auto-examples-howto-plot-optimize-evoked-py) as is. I'm getting the following error:

`TypeError: optimize_evoked() missing 2 required positional arguments: 'target_dpl' and 'initial_dpl'
`

I haven't modified any of the code so I'm passing all the required parameters. Any idea what this could mean?

Thanks

**Comment by jasmainak** [comment]
What version of `hnn-core` are you running? what do you get when you do: 

```py
print(hnn_core.__version__)
```

**Comment by carolinafernandezp** [comment]
0.3.dev0

**Comment by jasmainak** [comment, marked as answer]
You're looking at an old example. This is the right documentation for your version:

https://jonescompneurolab.github.io/hnn-core/dev/auto_examples/howto/optimize_evoked.html#sphx-glr-auto-examples-howto-optimize-evoked-py

You can select the version from the top right in the website 

![image](https://user-images.githubusercontent.com/15852194/207157001-f84c4107-7638-49f8-9b9f-ae34bc6fe63c.png)
...

**Comment by carolinafernandezp** [comment]
Thanks!
